### DISCUSSION TITLE
How to rerun the  Dynamic workflow  containing http tasks with different inputparameters  ?

### DISCUSSION THREAD

**Discussion by vidhi-pavan-agrawal** [Netflix/conductor#2522] [Q&A]
This is my workflow definition
{
  "name": "my_adhoc_unregistered_workflow",
  "workflowDef": {
    "ownerApp": "my_owner_app",
    "ownerEmail": "my_owner_email@test.com",
    "createdBy": "my_username",
    "name": "my_adhoc_unregistered_workflow",
    "description": "Test Workflow setup",
    "version": 1,
    "tasks": [
        {
            "name": "fetch_data",
            "type": "HTTP",
            "taskReferenceName": "fetch_data",
            "inputParameters": {
              "http_request": {
                "connectionTimeOut": "3600",
                "readTimeOut": "3600",
                "uri": "${workflow.input.uri}",
                "method": "GET",
                "accept": "application/json",
                "content-Type": "application/json",
                "headers": {
                }
              }
            },
            "taskDefinition": {
                "name": "fetch_data",
                "retryCount": 0,
                "timeoutSeconds": 3600,
                "timeoutPolicy": "TIME_OUT_WF",
                "retryLogic": "FIXED",
                "retryDelaySeconds": 0,
                "responseTimeoutSeconds": 3000
            }
        }
    ],
    "outputParameters": {
    }
  },
  "input": {
    "uri": "http://www.google.com"
  }
}

I want to rerun the workflow with some new values of the input parameters example "https://netflix.github.io"
Is there any way I should do it ( without using the rerun option on conductor UI: 5000 )?

**Comment by kishorebanala** [comment]
Hey @vidhi-pavan-agrawal, Not sure I follow.. would like to start another workflow with https://netflix.github.io/ as input? If so, you can do so from start workflow endpoint of the API. While you can try to update the input with rerun, they're usually not supposed to be used with different inputs.
